core/domain/stats_jobs_one_off_test.py

```diff
@@ -899,6 +1029,245 @@ def test_standard_operation(self):
         self.assertEqual(model.num_completions_v2, 3)
 
 
+class StatisticsAuditTests(OneOffJobTestBase):
+
+    ONE_OFF_JOB_CLASS = stats_jobs_one_off.StatisticsAudit
+    EXP_ID = 'EXP_ID'
+    EXP_VERSION = 1
+    STATE_NAME = 'state_1'
+
+    def setUp(self):
+        super(StatisticsAuditTests, self).setUp()
+
+        state_count = {
+            'state_name': {
+                'first_entry_count': 1,
+                'total_entry_count': 1,
+                'no_answer_count': 1
+            }
+        }
+        stats_models.ExplorationAnnotationsModel.create(
+            'exp_id1', '1', -2, -1, state_count)
+        stats_models.StateCounterModel.get_or_create('exp_id1', 'state_name')
+
+    def test_statistics_audit(self):
+        job_output = self.run_one_off_job()
+        self.assertEqual(
```

Comment:
This is testing too many things. Can we split them up to test each individual item? Same below.

Reply:
Done